dspace-api/src/main/java/org/dspace/workflow/WorkflowManager.java

```diff
@@ -412,7 +417,7 @@ public static boolean advance(Context c, WorkflowItem wi, EPerson e,
             // advance(...) will call itself if no workflow step group exists
             // so we need to check permissions only if a workflow step group is
             // in place.
-            if (wi.getCollection().getWorkflowGroup(1) != null)
+                if (wi.getCollection().getWorkflowGroup(1) != null && !wi.getCollection().getWorkflowGroup(1).isEmpty())
```

Comment:
Why did you change the indentation?

Reply:
@AlexanderS Good catch, I hadn't noticed this since I had my IDE configured to ignore whitespace/new line changes (And I'd copied this code from a test instance, incorrectly checking the indentation)